<launch>

<node name="rviz" pkg="rviz" type="rviz" args="-d $(find deminer)/config/rviz_publishpoint.rviz"/>

<arg name="map_file" value="$(find deminer)/maps/hall_of_fame.yaml"> </arg>	
<node pkg="map_server" type="map_server" name="map_server" args="$(arg map_file)"/>

<node name="playing_sound_node" pkg="sound_play" type="soundplay_node.py" output="screen"/>

<node name= "click_point" type="click_point" pkg="deminer" output="screen" launch-prefix="gnome-terminal --command"/>

</launch>


